be/src/vec/olap/olap_data_convertor.h

```diff
@@ -46,13 +46,16 @@ class IOlapColumnDataAccessor {
 
 class OlapBlockDataConvertor {
 public:
+    OlapBlockDataConvertor() = default;
```

Comment:
why not use OlapBlockDataConvertor(const TabletSchema* tablet_schema)?

Reply:
`OlapBlockDataConvertor(const TabletSchema* tablet_schema)` will insert all columns in `tablet_schema` into `_convertors`, we only need columns in `_alter_inverted_indexs`, if use `OlapBlockDataConvertor(const TabletSchema* tablet_schema)`, will assert failed in function `set_source_content`